packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/advanced-duration.jsx

```diff
@@ -66,9 +66,10 @@ const AdvancedDuration = ({
             {expiry_type === 'duration' ? (
                 <>
                     <div className='duration-container'>
-                        {duration_units_list.length > 1 && (
+                        {duration_units_list.length >= 1 && (
                             <Dropdown
-                                classNameDisplay='dc-dropdown__display--no-symbol'
```

Comment:
You should remove the scss code for `--no-symbol` if you've removed it from here and not using it in other places.

Reply:
sure, good remark.